optuna/trial.py

```diff
@@ -409,6 +426,20 @@ def _suggest(self, name, distribution):
         param_value_in_internal_repr = self.study.sampler.sample(self.storage, self.study_id, name,
                                                                  distribution)
 
+        return self._save_new_param_or_load_existing(name, param_value_in_internal_repr,
+                                                     distribution)
+
+    def _inject(self, name, param_value, distribution):
+        # type: (str, Any, distributions.BaseDistribution) -> Any
+
+        param_value_in_internal_repr = distribution.to_internal_repr(param_value)
+
+        return self._save_new_param_or_load_existing(name, param_value_in_internal_repr,
+                                                     distribution)
+
+    def _save_new_param_or_load_existing(self, name, param_value_in_internal_repr, distribution):
```

Comment:
How about renaming this to `_set_new_param_or_load_existing` for consistency?

Reply:
Thank you for your suggestion.
As we discussed offline, I will make two changes to keep both consistency and readability as follows:
-  `Trial._save_new_param_or_load_existing` is renamed to `Trial._set_new_param_or_get_existing`, and
- `Trial._inject` is removed.